tempesta_fw/t/functional/run_tests.py

```diff
@@ -110,6 +119,47 @@ def usage():
 # Install Ctrl-C handler for graceful stop.
 unittest.installHandler()
 
+
+#
+# Discover tests, configure environment and run tests
+#
+
+# For the sake of simplicity, Unconditionally discover all tests and filter them
+# afterwards instead of importing individual tests by positive filters.
+loader = unittest.TestLoader()
+tests = []
+shell.testsuite_flatten(tests, loader.discover('.'))
+
+
+# the default value of fs.nr_open
+nofile = 1048576
+resource.setrlimit(resource.RLIMIT_NOFILE, (nofile, nofile))
```

Comment:
Somewhere above root privileges are checked. I think these two code fragments should be grouped together to make cleaner why we request root privileges.

Reply:
fixed